core-server/src/main/java/org/orbisgis/orbisserver/manager/WpsServerManager.java

```diff
@@ -121,16 +132,26 @@ private void simpleWpsRequest() throws JAXBException {
         marshaller.marshal(factory.createGetCapabilities(getCapabilitiesType), out);
         //Write the OutputStream content into an Input stream before sending it to the wpsService
         InputStream in = new DataInputStream(new ByteArrayInputStream(out.toByteArray()));
-        ByteArrayOutputStream xml = (ByteArrayOutputStream) wpsServer.callOperation(in);
+        ByteArrayOutputStream xml = (ByteArrayOutputStream) this.getWpsServer().callOperation(in);
         //Get back the result of the DescribeProcess request as a BufferReader
         ByteArrayInputStream resultXml = new ByteArrayInputStream(xml.toByteArray());
         //Unmarshall the result and check that the object is the same as the resource unmashalled xml.
         Object resultObject = unmarshaller.unmarshal(resultXml);
-        WPSCapabilitiesType wpsCapabilitiesType = (WPSCapabilitiesType)((JAXBElement)resultObject).getValue();
+        WPSCapabilitiesType wpsCapabilitiesType = (WPSCapabilitiesType) ((JAXBElement) resultObject).getValue();
+        this.wpsCapabilitiesType = wpsCapabilitiesType;
         List<ProcessSummaryType> list = wpsCapabilitiesType.getContents().getProcessSummary();
-        for(ProcessSummaryType processSummaryType : list){
-            System.out.println(processSummaryType.getTitle().get(0).getValue());
+        for (ProcessSummaryType processSummaryType : list) {
+            processesList = processesList + processSummaryType.getTitle().get(0).getValue() + "\n";
         }
+        return processesList;
     }
 
+
+    /**
+     * Return the wpsCapabilitiesType object which is a xml object.
+     * @Return the wpsCapabilitiesType object.
+     */
+    public WPSCapabilitiesType getWPSCapabilitiesType() {
+        return this.wpsCapabilitiesType;
```

Comment:
So the `GetXMLFromGetCapabilities()` method returns the string list, and might be rename, and the `GetWPSCapabilitiesType()` method can return the WPSCapabilitiesType Object, is that ok ?